scala/Dummy.java

```diff
@@ -0,0 +1,5 @@
+public class Dummy {
+	public static void main(String[] args) {
```

Comment:
why add this? Just seems like more code in the class file?

Reply:
You're right, I removed the method.